jbmc/src/java_bytecode/remove_exceptions.cpp

```diff
@@ -489,7 +489,7 @@ void remove_exceptionst::instrument_exceptions(
     if(instr_it->is_decl())
     {
       code_declt decl=to_code_decl(instr_it->code);
-      locals.push_back(decl.symbol());
+      locals.push_back(to_symbol_expr(decl.symbol()));
```

Comment:
Aren't changes in this file in the wrong PR? Those don't seem to relate to `code_deadt` much? And in that other PR (on `code_declt`) this particular change wouldn't be necessary.

Reply:
Yes, when both PRs are put together, the need for this change will go away.
I'll adjust depending on which gets merged first.